cmd/agent/app/flare.go

```diff
@@ -53,6 +62,10 @@ var flareCmd = &cobra.Command{
 			return fmt.Errorf("unable to set up global agent configuration: %v", err)
 		}
 
+		if err := util.SetAuthToken(); err != nil {
```

Comment:
What is the reason for this change?

Reply:
This is necessary to call agent api to change runtime settings.

Reply:
A user without an access to the auth token can now run `agent flare`, with this change it will not be possible. I am not sure this is what we want.

Reply:
https://github.com/DataDog/datadog-agent/pull/8360/commits/e165515386ddef90398c5d1545ca9c3acc49a3e1